app/models/user.rb

```diff
@@ -1,5 +1,8 @@
+
 class User < ApplicationRecord
-  
+  attr_accessor :activation_token
+  before_save :downcase_email   
```

Comment:
I would move this piece of code near the other callback on line 27. This makes it easier to find all the callbacks / validations / etc.